playbooks/aws/openshift-cluster/provision.yml

```diff
@@ -10,14 +10,16 @@
     debug:
       msg: "openshift_aws_region={{ openshift_aws_region | default('us-east-1') }}"
 
-- import_playbook: provision_s3.yml
+#- import_playbook: provision_s3.yml
```

Comment:
I don't think you meant to comment out all these things.

Reply:
yup.  i'll need another commit to remove them.